athena.tf

```diff
@@ -37,16 +37,16 @@ resource "aws_athena_named_query" "create_view" {
   workgroup = aws_athena_workgroup.shepherd[count.index].id
   database  = split(":", aws_glue_catalog_database.shepherd[count.index].id)[1]
   query     = <<-EOT
-CREATE OR REPLACE VIEW shepherd_all
-AS
-%{for index, db in local.database_names~}
-SELECT * FROM ${db}.${local.table_name}
-%{if index < length(local.database_names) - 1~}
-UNION ALL
-%{endif~}
-%{endfor~}
-GO
-EOT
+      CREATE OR REPLACE VIEW shepherd_all
+      AS
+      %{for index, db in local.database_names~}
+      SELECT * FROM ${db}.${local.table_name}
+      %{if index < length(local.database_names) - 1~}
+      UNION ALL
+      %{endif~}
+      %{endfor~}
+      GO
+  EOT
```

Comment:
There isn't really a reason to modify the spacing here unless you want it to appear with spaces in the query when we update it with terraform. When using the `<<-EOT` syntax the `-` should be handling the whitespace stripping for us but only if the last `EOT` is on the same line as the rest of the code. So I suggest you revert this change.

Reply:
I returned EOT section to flush left.